apps/condo/domains/banking/components/BankAccountReport.tsx

```diff
@@ -138,11 +138,11 @@ const BankAccountReportContent: IBankReportContent = ({ bankAccountReports = [],
     const router = useRouter()
 
     const [activeTab, setActiveTab] = useState(get(bankAccountReports, '0.data.categoryGroups.0.id'))
-    const [selectedPeriod, setSelectedPeriod] = useState(0)
+    const [selectedPeriod, setSelectedPeriod] = useState(get(router, 'query.period') || get(bankAccountReports, '0.period'))
```

Comment:
Don't we need a short documentation for page query params?
Parsing of url query is located not at page level, but in components and it will be hard to figure out at first look.